backend/app/resources/api/v1/transportation_planning_factor_resource.rb

```diff
@@ -2,7 +2,7 @@ module Api
   module V1
     class TransportationPlanningFactorResource < BaseResource
       model_name 'TransportationPlanningFactors'
-      
+
```

Comment:
@bfreeds can you help us figure out why we can't remove this from the diff?

Reply:
@allthesignals I looked with @trbmcginnis and I'm not sure what's happening here - I think that we should just deal with this odd changeset in this PR anticipating that the spacing is going to be formatted automatically as soon as we get prettier setup.